asserts/ifacedecls.go

```diff
@@ -341,8 +352,9 @@ var (
 )
 
 // Check checks whether attrs don't match the constraints.
-func (c *AttributeConstraints) Check(attrs map[string]interface{}, ctx AttrMatchContext) error {
-	return c.matcher.match("", attrs, ctx)
+func (c *AttributeConstraints) Check(attrer interfaces.Attrer, ctx AttrMatchContext) error {
+	var nm map[string]interface{}
```

Comment:
What is the point of this map since it's not returned anywhere? Can you just pass nil to the function below?

Reply:
Done.